Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -178,13 +178,34 @@ void OpenALStream::SoundLoop()
   surround_capable = false;
 #endif
 
-  u32 ulFrequency = m_mixer->GetSampleRate();
-  numBuffers = SConfig::GetInstance().iLatency + 2;  // OpenAL requires a minimum of two buffers
+  u32 ul_frequency = m_mixer->GetSampleRate();
+  // OpenAL requires a minimum of two buffers
+  num_buffers = SConfig::GetInstance().iNumBuffers;
```

Comment:
I don't understand. Why do you want to be able to change the number of buffers? The way I understand it, you generally want the minimum number of buffers that the hardware/software will support.

What you're saying about SoundTouch doesn't make sense to me. SoundTouch wants larger buffers, not more buffers, because its output size is adjusted based on its configuration. Larger buffers = higher quality shifting.